From 64ed7266c5545f935bf9527066b0204d09e43b21 Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Tue, 4 Aug 2020 19:28:48 +0800
Subject: [PATCH] libcjson.pc.in: add prefix/execute_prefix definitions

Add prefix/execute_prefix definitions, because OpenWrt pkg-config
will replace the definitions to staging target rootfs during checking.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
---
 library_config/libcjson.pc.in | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/library_config/libcjson.pc.in b/library_config/libcjson.pc.in
index 157a465..bc504fd 100644
--- a/library_config/libcjson.pc.in
+++ b/library_config/libcjson.pc.in
@@ -1,5 +1,7 @@
-libdir=@CMAKE_INSTALL_FULL_LIBDIR@
-includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
 
 Name: libcjson
 Version: @PROJECT_VERSION@
-- 
2.7.4

